net.technicpack.TechnicLauncher.metainfo.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<component type="desktop">
+    <id>net.technicpack.TechnicLauncher</id>
+    <metadata_license>MIT</metadata_license>
+    <name>Technic Launcher</name>
+    <summary>The Technic Launcher will get you playing your favorite modpacks in no time. Choose from thousands of modpacks and install them with a single click.</summary>
+    <description>
+    	<p>Stylish, Technic, The launcher you all remember, Technic Launcher.</p>
+        <p>Features:</p>
+        <ul>
+          <li>
+          Easy Modpack Installs
+          
+          It's as easy as clicking play. The Technic Launcher handles all of the heavy lifting for you. You will be playing your favorite modpack in minutes.
+          </li>
+          
+          <li>
+          Discover
+          
+          Explore the Technic Platform from right within the launcher. Find the most popular modpacks of the week and install them right then and there. Just like that you are playing an all new modpack!
+          </li>
+        </ul>
+    </description>
+   
+    <categories>
+        <category>Game</category>
+    </categories>
+    
+    <url type="homepage">https://www.technicpack.net</url>
+    <url type="bugtracker">https://forums.technicpack.net/</url>
+    <url type="help">https://www.technicpack.net/help</url>
+    <url type="contact">https://www.technicpack.net/contact-us</url>
+
+    <launchable type="desktop-id">net.technicpack.TechnicLauncher.desktop</launchable>
+
+    <project_license>GPL-3.0</project_license>
+
+    <developer_name>Syndicate</developer_name>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/Doomsdayrs/flathub/net.technicpack.TechnicLauncher/images/discover.png</image>
+        </screenshot>
+        <screenshot>
+            <image >https://raw.githubusercontent.com/Doomsdayrs/flathub/net.technicpack.TechnicLauncher/images/modpacks.png</image>
+        </screenshot>
+    </screenshots>
+    
+  <releases>
+   <release date="2021-01-25" version="4.709.0"/>
+  </releases>
+  
+  <content_rating type="oars-1.0">
```

Comment:
Not sure where you took it from, but 1.1 specification allows skipping attributes set to `none`.